build.gradle

```diff
@@ -96,6 +96,7 @@ allprojects {
             palantirJavaFormat('2.26.0')
             trimTrailingWhitespace()
             endWithNewline()
+            ratchetFrom('main')
```

Comment:
what does this do

Reply:
Making things break. :-/

This was apparently needed for Spotless to have some kind of "baseline" against which it should determine that a file was changed so it could know it needed to update copyright year. But I am not getting it to work. :( The "origin/main" worked locally for me, but not on the CI build here. I thought that a local reference might have worked, but apparently not. :(